clients/src/main/java/org/apache/kafka/clients/telemetry/TelemetrySubscription.java

```diff
@@ -100,4 +100,33 @@ public String toString() {
             .add("metricSelector=" + metricSelector)
             .toString();
     }
+
+    /**
+     * Create a new TelemetrySubscription object with the given {@code pushIntervalMs} parameter.
+     * @param pushIntervalMs time interval in millisecond of the next push
+     * @return a new TelemetrySubscription
+     */
+    public TelemetrySubscription alterPushIntervalMs(long pushIntervalMs) {
```

Comment:
Need to also update the `fetchMs` or else our new `pushIntervalMs` won't start from the correct point in time. Maybe pass in a value to the `alterPushIntervalMs` method or something?